package/rmx-only/package

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(rm1-only rm2-only)
+pkgdesc="Ensure packages are not installed on the wrong device"
+pkgver=0.1.0-1
+url=http://0.0.0.0/
+timestamp=2020-11-06T09:35Z
+section=meta
+maintainer="Linus K. <linus@cosmos-ink.net>"
+license=MIT
```

Comment:
We should probably changes this to be equal to the repo maintainers. These packages will by more of an first-party/internal one.

Reply:
toltec is MIT licensed, so I believe this is fine.